src/applications/mhv/secure-messaging/sass/navigation.scss

```diff
@@ -58,32 +65,41 @@
     border: 0px;
   }
   .sidebar-navigation-messages-list {
-    background-color: $color-gray-lightest;
     .sidebar-navigation-messages-list-header {
-      border-left: 5px solid;
+      &.is-active {
+        background-color: $color-gray-lightest;
+        border-left: 4px solid $color-base;
+      }
       font-weight: 700;
       a {
         padding-top: 7px;
+        color: $color-link-default;
+        &.is-active{
+          color: $color-black;
+        }
         &:active {
           color: $color-gray-dark;
           text-decoration: underline;
         }
       }
     }
     .sidebar-navigation-messages-list-menu {
-      padding-left: 16px;
       ul {
-        li:nth-child(1) {
+        // li:nth-child(1) {
+        //   a {
+        //     padding-top: 3px;
+        //     margin-top: 0px;
+        //   }
+        // }

```

Comment:
would you need to keep this for anything or can we remove these commented lines?